inc/Engine/Optimization/Minify/CSS/Minify.php

```diff
@@ -135,10 +135,10 @@ private function replace_url( $url ) {
 			$url = rocket_add_url_protocol( $url );
 		}
 
-		$unique_id     = md5( $url . $this->minify_key );
-		$filename      = preg_replace( '/\.(css)$/', '-' . $unique_id . '.css', ltrim( rocket_realpath( $parsed_url['path'] ), '/' ) );
-		$minified_file = rawurldecode( $this->minify_base_path . $filename );
-		$minify_url    = $this->get_minify_url( $filename, $url );
+		$filename            = ltrim( rocket_realpath( $parsed_url['path'] ), '/' );
+		$minified_file       = rawurldecode( $this->minify_base_path . $filename );
+		$minified_file_mtime = rocket_direct_filesystem()->mtime( $minified_file );
+		$minify_url          = add_query_arg( 'ver', $minified_file_mtime ? $minified_file_mtime : '', $this->get_minify_url( $filename, $url ) );
```

Comment:
How about using the `$this->minify_key` value in case `mtime()` returned false? That way we always have a query string value, instead of an empty one.